libraries/SD_MMC/examples/SDMMC_Test/SDMMC_Test.ino

```diff
@@ -172,8 +210,34 @@ void testFileIO(fs::FS &fs, const char * path){
 
 void setup(){
     Serial.begin(115200);
-    if(!SD_MMC.begin()){
-        Serial.println("Card Mount Failed");
+    while(!Serial) { delay(10); }
+    Serial.println("SDMMC_Test.ino starting!");
+
+    // If you are using any other ESP32-S3 board than ESP32-S3-USB-OTG which has preset default pins, you will
+    //    need to specify the pins with the following example of SD_MMC.setPins()
+    // If you want to use only 1-bit mode, you can use the line with only one data pin (d0) begin changed.
+    // Please note that ESP32 does not allow pin change and will always fail.
```

Comment:
As its not that true that the setPins() will always fail, because if you use the right pins its not gonna fail, but it don't make sense to call it as it won't make any difference. I would rather write that ESP32 has dedicated pins for SDMMC so there is no need to call setPin as it won't have any effect. Something like that :)